views/head.tpl.html

```diff
@@ -1,5 +1,5 @@
 <head>
-    <title>Wakapi – Coding Statistics</title>
+    <title>Wakapi - Coding Statistics</title>
```

Comment:
These "long" dashes were actually intended. Is there a good reason to replace them?

Reply:
I reverted this file, but happy to revert the `.go` error messages also if you want. 
An argument might be that it's more standard, but it's a rather weak reason.